Easydict/NewApp/View/SettingView/Tabs/DisabledAppTab.swift

```diff
@@ -133,15 +133,16 @@ private struct ListToolbar: View {
         VStack(spacing: 0) {
             Divider()
             HStack(spacing: 0) {
-                ListButton(imageName: "plus", enabled: true) {
+                ListButton(systemName: "plus") {
                     disabledAppViewModel.isImporting.toggle()
                 }
-                Divider().padding(.vertical, 1)
-                let isNoSelectedApps = disabledAppViewModel.selectedAppModels.isEmpty
-                ListButton(imageName: "minus", enabled: !isNoSelectedApps) {
+                .environment(\.isEnabled, true)
+                Divider()
+                    .padding(.vertical, 1)
+                ListButton(systemName: "minus") {
                     disabledAppViewModel.removeDisabledApp()
                 }
-                .disabled(isNoSelectedApps)
+                .environment(\.isEnabled, !disabledAppViewModel.selectedAppModels.isEmpty)
```

Comment:
Ouch, here we should use `.disabled` instead of `.environment`.

Reply:
Nice, lesson learned.